pom.xml

```diff
@@ -29,11 +36,22 @@
     <name>WildFly Elytron</name>
     <description>WildFly Security SPIs</description>
 
-    <parent>
-        <groupId>org.jboss</groupId>
-        <artifactId>jboss-parent</artifactId>
-        <version>16</version>
-    </parent>
+    <issueManagement>
+        <system>Jira</system>
+        <url>https://issues.jboss.org/browse/ELY</url>
+    </issueManagement>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
```

Comment:
As long as is up to date for the most recent tag I think it is fine, as long as it covers an issue we had recently where a user suddenly noticed the project being downloaded and didn't know where to do to look for the source I think it is fine.